### PR TITLE
Add page customizing-cryogen

### PR DESCRIPTION
**Pull request by holyjak**
Any comments / improvements? Perhaps move it elsewhere in the docs structure?

### CODE REVIEW

File: resources/templates/md/docs/customizing-cryogen.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{:title "Customizing Cryogen"
```
Comment (lacarmen) [inline resources/templates/md/docs/customizing-cryogen.md]: Maybe change to "Customizing/Extending Cryogen"?

File: resources/templates/md/docs/customizing-cryogen.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ :layout :page
+ :page-index 12
+ :section "Additional Resources"}
```
Comment (lacarmen) [inline resources/templates/md/docs/customizing-cryogen.md:5]: Should add `:toc` to this page :)

File: resources/templates/md/docs/customizing-cryogen.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+   that can leverage all the available `params` (the data eventually passed to Selmer) and the
+   provided `site-data` to derive modified `params`. See the code for the available data.
+
+### Extra pages / posts
+
+You can leverage the functions in `cryogen-core.compiler` and write your own, invoking them from your `cryogen.core/-main` and `cryogen.server/init`, to generate additional pages and posts in the same way as `compile-assets-timed` does.
+
+### Customizing the code
+
+You can copy the `cryogen-core.compiler` namespace directly into your project (where it will override the one from the cryogen-core.jar) and modify it to your liking. It is not very long or complicated and is quite easy to modify. That is [what we did for this site](https://github.com/cryogen-project/cryogen-docs/blob/fd601c857cc88f7cb633a41c47b4c692e1522ed8/src/cryogen/compiler.clj).
```
Comment (lacarmen) [inline resources/templates/md/docs/customizing-cryogen.md]: We should add a quick disclaimer at the end here: "Although this site is generated using a much older version of `cryogen-core`, you may still use the same strategy today"

I feel like people might wonder about that 😛

**Review by lacarmen — CHANGES_REQUESTED** [review]
Just need some minor changes.

This is a great start! We could always create a dedicated "Customizing Cryogen" heading if more options/tricks become available in the future.

**Comment by holyjak** [comment]
Comments addressed.

**Review by lacarmen — APPROVED** [review]
Awesome, looks great!

**Pull request merged**